pkg/jiraservicedesk/client/client_project.go

```diff
@@ -11,7 +12,8 @@ import (
 
 const (
 	// Endpoints
-	EndpointApiVersion3Project = "/rest/api/3/project"
+	EndpointApiVersion3Project   = "/rest/api/3/project"
+	EndpointCustomerAccessStatus = "/rest/servicedesk/1/servicedesk"
```

Comment:
`EndpointCustomerAccessStatus` is a generic endpoint which can have multiple further sub-paths. Please use a generic name here